.aws/aws_create_template.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# init script for quickly setting up an emx2 on a vanilla ubuntu, tested on xenial-16.04
+# we use this template in the aws_start_preview_from_build_script
+
+## create an instance
+aws ec2 run-instances --image-id ami-09042b2f6d07d164a --count 1 --instance-type t2.micro --key-name mswertz --security-group-ids sg-034a0713c918bb42f --subnet-id subnet-093e36ba6d1b5f420 --region eu-central-1 --instance-initiated-shutdown-behavior terminate
+
+# add repositories for postgresql and java
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
```

Comment:
Maybe use docker and our docker-compose to bootup the instance? That would simplify the configuration a lot.

Reply:
I considered that, but should then be able to pass the docker image version to docker compose, is that already possible?